code/game/gamemodes/bloodbrother/brother.dm

```diff
@@ -0,0 +1,174 @@
+	to_chat(owner.current, "The Syndicate only accepts those that have proven themselves. Prove yourself and prove your [team.member_name]s by completing your objectives together!")
+	owner.announce_objectives()
+	antag_memory += "<b>Your brothers are</b>: [brother_text].<br>"
+	give_meeting_area()
+
+/datum/antagonist/brother/proc/finalize_brother()
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/tatoralert.ogg', 100, FALSE, pressure_affected = FALSE)
+
+/datum/antagonist/brother/proc/update_brother_icons_added(datum/mind/brother_mind)
+	if(locate(/datum/objective/hijack) in owner.objectives)
+		var/datum/atom_hud/antag/hijackhud = GLOB.huds[ANTAG_HUD_TRAITOR]
+		hijackhud.join_hud(owner.current, null)
+		set_antag_hud(owner.current, "hudhijackbrother")
+	else
+		var/datum/atom_hud/antag/traitorhud = GLOB.huds[ANTAG_HUD_TRAITOR]
+		traitorhud.join_hud(owner.current, null)
+		set_antag_hud(owner.current, "hudbrother")
+
+/datum/antagonist/brother/proc/update_brother_icons_removed(datum/mind/brother_mind)
+	var/datum/atom_hud/antag/traitorhud = GLOB.huds[ANTAG_HUD_TRAITOR]
+	traitorhud.leave_hud(owner.current, null)
+	set_antag_hud(owner.current, null)
+
+
+/datum/team/brother_team
+	name = "brotherhood"
+	member_name = "Blood brother"
+	var/meeting_area
+	var/static/meeting_areas = list("The Bar","Dorms","Escape Dock","Arrivals","Holodeck","Primary Tool Storage","Recreation Area","Chapel","Library")
+	var/list/assigned_targets = list() // This includes assassinate as well as steal objectives. prevents duplicate objectives
+
+/datum/team/brother_team/is_solo()
+	return FALSE
+
+/datum/team/brother_team/Destroy()
+	for(var/datum/mind/brother in members)
+		if(brother.has_antag_datum(/datum/antagonist/brother))
+			brother.remove_antag_datum(/datum/antagonist/brother)
+	SSticker.mode.brother_teams -= src	
+	return ..()
+
+/datum/team/brother_team/proc/pick_meeting_area()
+	meeting_area = pick(meeting_areas)
+	meeting_areas -= meeting_area
+
+/datum/team/brother_team/proc/update_name()
+	var/list/last_names = list()
+	for(var/datum/mind/M in members)
+		var/list/split_name = splittext(M.name," ")
+		last_names += split_name[split_name.len]
+
+	name = last_names.Join(" & ")
+
+/datum/team/brother_team/proc/add_objective(datum/objective/O) // pre team set up.
+	objectives += O
+
+/datum/team/brother_team/proc/forge_brother_objectives()
+	objectives = list()
+	var/is_hijacker = prob(10)
+	for(var/i = 1 to max(1, (config.brother_objectives_amount + (members.len >2) - is_hijacker)))
+		forge_single_objective()
+	if(is_hijacker)
+		if(!locate(/datum/objective/hijack) in objectives)
+			add_objective(new/datum/objective/hijack)
+	else if(!locate(/datum/objective/escape) in objectives)
+		add_objective(new/datum/objective/escape)
+
+/datum/team/brother_team/proc/forge_single_objective()
+	if(prob(50))
+		var/list/active_ais = active_ais()
+		if(active_ais.len && prob(100/GLOB.player_list.len))
```

Comment:
It's not uncommon for the population to go above 100 so I think this would only harm objective variety.

Reply:
yeah could move the objective down or add a minimum argument.